src/Auth0.OidcClient.WPF/WebViewBrowser.cs

```diff
@@ -53,6 +53,7 @@ public async Task<BrowserResult> InvokeAsync(BrowserOptions options, Cancellatio
             var tcs = new TaskCompletionSource<BrowserResult>();
 
             var window = _windowFactory();
+            #pragma warning disable 618
```

Comment:
WebViewCompatible is obsolete, but we have no alternative on old platforms. For newer platforms, we already use the new version.

Ignoring this warning.